benches/main.rs

```diff
@@ -0,0 +1,272 @@
+#![cfg(feature = "benches")]
+#![feature(test)]
```

Comment:
Could this file be renamed to perhaps `spec.rs` or something that indicates that it's benchmarking the spec test suite?

FWIW the spec test suite I don't think is really the most interesting of benchmarks, but at least for our own internal tracking and performance monitoring it probably isn't so bad to track!